library/src/main/java/io/constructor/data/DataManager.kt

```diff
@@ -185,4 +187,65 @@ constructor(private val constructorApi: ConstructorApi, @ConstructorSdk private
     fun trackRecommendationResultsView(recommendationResultViewRequestBody: RecommendationResultViewRequestBody, params: Array<Pair<String, String>> = arrayOf()): Completable {
         return constructorApi.trackRecommendationResultsView(recommendationResultViewRequestBody, params.toMap())
     }
+
+    fun getNextQuestion(quizId: String, encodedParams: Array<Pair<String, String>> = arrayOf(), preferencesHelper: PreferencesHelper): Observable<ConstructorData<QuizResponse>> {
+        val scheme = preferencesHelper.scheme
+        val serviceUrl = preferencesHelper.quizzesServiceUrl
+        val url = "${scheme}://${serviceUrl}/${ApiPaths.URL_QUIZ_NEXT_QUESTION.format(quizId)}${getAdditionalParamsQueryString(encodedParams)}"
```

Comment:
This deviates a bit from the normal pattern where we have a dynamic url. In this case, we specify an absolute URL to get around the default service url (`ac.cnstrc.com`) that's used.